src/commands/settings/handlers/privateKeyHandler.ts

```diff
@@ -0,0 +1,28 @@
+import { getPrivateKey, setPrivateKey } from '../../../lib/keys.js';
+import { confirmPrompt, privateKeyPrompt } from '../../../lib/prompts.js';
+import { viewHandler } from './viewHandler.js';
+
+export const privateKeyHandler: (
+  ...args: unknown[]
+) => void | Promise<void> = async (privateKey: string) => {
+  const existingPrivateKey = await getPrivateKey();
+
+  // 2.1 Is there an existing private key?
+  if (existingPrivateKey) {
+    // 2.2 Prompt the user to overwrite the existing private key
+    (await confirmPrompt('Overwrite existing Private Key?'))
```

Comment:
"Seems like you already have a private key stored. Are you sure you want to override it with a new one?"